impeller/entity/contents/conical_gradient_contents.cc

```diff
@@ -51,18 +51,6 @@ void ConicalGradientContents::SetFocus(std::optional<Point> focus,
   focus_radius_ = radius;
 }
 
-bool ConicalGradientContents::IsOpaque() const {
```

Comment:
Was this removed for the cases where the gradient doesn't occupy the full square?  That wasn't the crux of the reported issue, right?  I'm not sure if it's worth detecting those scenarios for IsOpaque.

Reply:
@bdero are you fine with us opting out of IsOpaque?  I think we could calculate this by expanding on what we previously had if it's worth it.

Reply:
Yeah this is just used to optimize the blend mode for HSR and prevent unnecessary overdraw, so it's safe to just remove for now if it's wrong.